unstructured_ingest/v2/processes/partitioner.py

```diff
@@ -189,14 +190,16 @@ async def partition_via_api(
         self, filename: Path, metadata: Optional[dict] = None, **kwargs
     ) -> list[dict]:
         from unstructured_client import UnstructuredClient
+        from unstructured_client.models.operations import PartitionRequest
 
         logger.debug(f"partitioning file {filename} with metadata: {metadata}")
         client = UnstructuredClient(
             server_url=self.config.partition_endpoint,
             api_key_auth=self.config.api_key.get_secret_value(),
         )
         partition_params = self.create_partition_parameters(filename=filename)
-        resp = await self.call_api(client=client, request=partition_params)
+        partition_request = PartitionRequest(partition_params)
```

Comment:
You could be more consistent and name this object either `partition_request` or `partition_req_obj` in both files.

Reply:
I had auto merge on, but agreed! I'll keep this in mind when we're in here next.